org.hitchhiker_linux.vapad.json

```
@@ -0,0 +1,41 @@
+{
+    "app-id" : "org.hitchhiker_linux.vapad",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "vapad",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "vapad",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://codeberg.org/jeang3nie/vapad",
+                    "tag" : "v0.5.0"
+                    "commit" : "a13da260026cdccc57cc8a1ca1ebde94e961f833"
```

```suggestion
                    "tag" : "v0.5.0",
                    "commit" : "a13da260026cdccc57cc8a1ca1ebde94e961f833"
```

Also looking at the diff, you add a commit on top of a tag (good) but say it's because it's a new change. Did you retag (please don't)?

---

I did retag, but will refrain in the future. Apologies, this is my first time with this process.